config/tailwind/buttons.js

```diff
@@ -4,7 +4,7 @@ export default plugin(({ addComponents }) => {
   // Base Styles
   const base = {
     // core
-    '@apply inline-flex items-center rounded font-bold transition text-base gap-12 cursor-pointer no-underline':
+    '@apply inline-flex items-center justify-center rounded font-bold transition text-base gap-12 cursor-pointer no-underline':
```

Comment:
This addition makes sure that text is fully centered when the element is "squared" for the pagination links.